source/features/i18n/translations/de.json

```diff
@@ -135,16 +135,16 @@
     "performance": "Performance",
     "profitMargin": "Gewinnspanne",
     "ranking": "Rang",
-    "retiring": "!!!Retiring in",
+    "retiring": "Ruhestand in",
     "stakePoolsSearch": "Nach einem bestimmten Stake Pool suchen",
     "stakePoolsTitle": "Stake Pools",
-    "unmoderated": "!!!You have consented to view unmoderated stake pool data.",
-    "unmoderatedContent": "!!!Do you want to see unmoderated content?",
-    "unmoderatedEntitiesControl": "!!!This information is provided and hosted by stake pool operators and it is not in control of entities running this website.",
-    "unmoderatedLeave": "!!!Leave this page",
-    "unmoderatedPoolTickerSymbols": "!!!Stake pool ticker symbols, titles, descriptions and websites are not moderated and may contain inappropriate content.",
-    "unmoderatedStakePoolData": "!!!Stake pool data is not moderated",
-    "unmoderatedWarning": "!!!Warning:"
+    "unmoderated": "Sie haben zugestimmt, nicht moderierte Einsatzpooldaten anzuzeigen.",
```

Comment:
"Einsatzpooldaten" is not a real word 😅 please replace it with "Stake-Pool Daten"